src/content/docs/en/core-concepts/routing.mdx

```diff
@@ -219,6 +220,74 @@ const { title, text } = page;
 </html>
 ```
 
+## Redirects (experimental)
+
```

Comment:
```suggestion
## Redirects

Sometimes you will need to redirect your readers to a new page, either permanently because your site structure has changed, or in response to an action such as logging in to an authrenticated route.

You can define rules to [redirect users to permanently-moved pages](#configured-redirects-experimental) in your Astro config. Or, [redirect users dynamically](#dynamic-redirects) as they use your site.

### Configured Redirects (experimental)
```
Especially since only the new (permanent, configured) redirects are actually experimental, I think for now this sectioning makes sense? Both types of redirects are sub-headings of redirects?

Reply:
Yeah, good point, I like this.